docs/source/markdown/options/cap-add.md

```diff
@@ -5,3 +5,11 @@
 #### **--cap-add**=*capability*
 
 Add Linux capabilities.
+
+Granting additional capabilities increases the privileges of the
+processes running inside the container and potentially allow it to
+break out of confinement.  Capabilities like `CAP_SYS_PTRACE`,
+`CAP_MKNOD` and `CAP_SYS_MODULE` are particularly dangerous.
+
+Before adding any capability, review its security implications and
+ensure it is really necessary for the container’s functionality.
```

Comment:
I guess it would be good to mention `capabilities(7)` and `user_namespaces(7)` as reference here, user_namespaces because I saw a few people confused why the caps they were adding don't work when running rootless, i.e. CAP_MKNOD and CAP_SYS_MODULE do not work in a user namesapce.

Reply:
what do you think of the new version?